extension/templates/devTools/activity/graph.js

```diff
@@ -0,0 +1,90 @@
+define(['d3', 'util/Radio'], function(d3, Radio) {
+  
+  return {
+
+    formatData: function(data) {
+
+      var rectHeight = 20;
```

Comment:
I think we should `deep` clone data when it comes down so that we don't mutate the activity...

I'm not sure if we have a deep clone utility but, it's basically `$.extend({}, data, true)` i think